thunder/tests/opinfos.py

```diff
@@ -542,6 +542,23 @@ def _fn(x):
     return _fn
 
 
+@wraps(torch.Tensor.is_cuda)
+def _is_cuda_torch(x: torch.Tensor):
+    # Should Number be accepted?
+    return x.is_cuda
+
+
+is_cuda_opinfo = OpInfo(
+    ltorch.is_cuda,
```

Comment:
This makes a lot of sense, but what if a helper that access the property was used here and for the `torch_reference`? That way the OpInfo would test that the property was registered properly. Something like

```
def _is_cuda_property(t) -> bool:
  return t.is_cuda
```

and then

```
is_cuda_opinfo = OpInfo(
  _is_cuda_property,
  ...,
  torch_reference=_is_cuda_property,
  ...
)
```

Reply:
I am trying to wrap my head around your comment but I am not sure if I agree with your point. Is the main purpose of `OpInfo` to test the registration of the property? When referring to other `OpInfo`s in `tests/opinfos.py`, I am seeing that `op` usually refers to the `ltorch` implementation. Is there a reason why you would want this particular case to have the helper function to be assigned for both `op` and `torch_reference`?
Maybe I have misunderstood the point of `OpInfo` in the first place..
Let me know if I misunderstood anything and I would love to learn more about your opinion!

Reply:
This is certainly an unusual case. I think in this case we are testing that the property works as expected. This is unlike existing OpInfos, which are comparing one operation with another. 

Because the property is accessed the same way in thunder and in PyTorch, we can use the same testing helper to acquire it.

Reply:
That makes sense! Thanks for the detailed explanation. Will update this as well :)

Reply:
Hmm, simply replacing `ltorch.is_cuda` to the helper function is throwing an error. Let me look into this more carefully!